### PR TITLE
Updated sudoers file 

### PR DESCRIPTION
**Pull request by shyam-biradar**

### CODE REVIEW

**Comment by shyam-biradar** [comment]
@amitkumar-a4  Please review the sudoers file change.

File: redhat-director-scripts/docker/trilio-datamover/nova-sudoers [hunk 1]
```diff
@@ -1 +1 @@
-nova        ALL=(ALL)       NOPASSWD: ALL
+nova ALL = (root) NOPASSWD: /home/tvault/.virtenv/bin/privsep-helper *
```
Comment (amitkumar-a4) [inline redhat-director-scripts/docker/trilio-datamover/nova-sudoers]: In RHOSP this path is same?

Comment (shyam-biradar) [reply redhat-director-scripts/docker/trilio-datamover/nova-sudoers]: It should be no different than normal redhat/centos setup.

Comment (amitkumar-a4) [reply redhat-director-scripts/docker/trilio-datamover/nova-sudoers]: If you are sure then ok.

**Pull request merged**